examples/vm_update.yml

```diff
@@ -26,56 +26,66 @@
         remove_disk_uuid: ""
         subnet_uuid: ""
 
-  - name: Update VM
-    ntnx_vms:
-      vm_uuid: "{{ vm_uuid }}"
-      name: updated
-      desc: updated
-      categories:
-        AppType:
-          - Apache_Spark
-      disks:
-        - type: "DISK"
-          clone_image:
-            name: "{{ image_name }}"
-          bus: "SCSI"
-          size_gb: 20
-        - type: DISK
-          size_gb: 3
-          bus: PCI
-        - type: DISK
-          size_gb: 1
-          bus: SCSI
-          storage_container:
-            uuid: "{{ storage_container_uuid }}"
-      networks:
-        - is_connected: true
-          subnet:
-            uuid: "{{ network_dhcp_uuid }}"
-        - is_connected: false
-          subnet:
-            uuid: "{{ static.uuid }}"
-          private_ip: "{{ network_static_ip }}"
-    register: result
+    - name: Update VM
+      ntnx_vms:
+        vm_uuid: "{{ vm_uuid }}"
+        name: updated
+        desc: updated
+        categories:
+          AppType:
+            - Apache_Spark
+        disks:
+          - type: "DISK"
+            clone_image:
+              name: "{{ image_name }}"
+            bus: "SCSI"
+            size_gb: 20
+          - type: DISK
+            size_gb: 3
+            bus: PCI
+          - type: DISK
+            size_gb: 1
+            bus: SCSI
+            storage_container:
+              uuid: "{{ storage_container_uuid }}"
+        networks:
+          - is_connected: true
+            subnet:
+              uuid: "{{ network_dhcp_uuid }}"
+          - is_connected: false
+            subnet:
+              uuid: "{{ static.uuid }}"
+            private_ip: "{{ network_static_ip }}"
+      register: result
 
-  - name: Update VM by deleting and editing disks and subnets
-    ntnx_vms:
-      vm_uuid: "{{ vm_uuid }}"
-      name: update diks
-      desc: update disks
-      disks:
-        - type: "DISK"
-          uuid: "{{ disk_uuid }}"
-          size_gb: 30
-        - state: absent
-          uuid: "{{ remove_disk_uuid }}"
-      networks:
-        - state: absent
-          uuid: "{{ subnet_uuid }}"
-    register: result
+    - name: Update VM by cloning image into CD ROM
```

Comment:
Create a seperate example file where you create empty CD ROM and then update it.

You are updating a VM which is supposed to have CD ROM empty. Lets have a clear example and not use this current for our usecase